resources/functionMap.php

```diff
@@ -6794,9 +6795,9 @@
 'MongoDB\Driver\ReadPreference::getMode' => ['int'],
 'MongoDB\Driver\ReadPreference::getTagSets' => ['array'],
 'MongoDB\Driver\Server::__construct' => ['void', 'host'=>'string', 'port'=>'string', 'options='=>'array', 'driverOptions='=>'array'],
-'MongoDB\Driver\Server::executeBulkWrite' => ['', 'namespace'=>'string', 'zwrite'=>'BulkWrite'],
-'MongoDB\Driver\Server::executeCommand' => ['MongoDB\Driver\Cursor', 'db'=>'string', 'command'=>'MongoDB\Driver\Command', 'readPreference='=>'MongoDB\Driver\ReadPreference'],
-'MongoDB\Driver\Server::executeQuery' => ['MongoDB\Driver\Cursor', 'namespace'=>'string', 'query'=>'MongoDB\Driver\Query', 'readPreference='=>'MongoDB\Driver\ReadPreference'],
+'MongoDB\Driver\Server::executeBulkWrite' => ['', 'namespace'=>'string', 'bulkWrite'=>'BulkWrite', 'options='=>'array|MongoDB\Driver\WriteConcern|nullarray|MongoDB\Driver\WriteConcern|null'],
```

Comment:
I think the `bulkWrite` argument needs a FQCN. There's also a missing return type

```suggestion
'MongoDB\Driver\Server::executeBulkWrite' => ['MongoDB\Driver\WriteResult', 'namespace'=>'string', 'bulkWrite'=>'MongoDB\Driver\BulkWrite', 'options='=>'array|MongoDB\Driver\WriteConcern|null'],
```

Reply:
Fixed